app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -73,5 +81,20 @@ def default_update_type
     def specialist_sectors
       [edition.primary_specialist_sector_tag].compact + edition.secondary_specialist_sector_tags
     end
+
+    def access_limited
+      {
+        users: users.map(&:id)
+      }
+    end
+
+    def users
+      return @users if @users
+
+      org_users = edition.organisations.map(&:users).flatten
+      all_users = (org_users + edition.authors)
```

Comment:
I don't believe it is the case that authors can see access_limited content, even if they’re not in the org. `can_see?` only checks organisation membership, and it appears to be possible to create a document associated to another org, access limit it, and be prevented from seeing it.